.github/workflows/tests.yml

```diff
@@ -11,8 +11,8 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      TF_VERSION: 2.2
-      TORCH_VERSION: 1.5
+      TF_VERSION: 2.3
+      TORCH_VERSION: 1.6

```

Comment:
Do we ever want to test against earlier versions of TF or Torch, rather than just the latest?

Reply:
Ideally, it would be nice to test a much larger number of dependency permutations, but it gets prohibitive. Especially when we are using the free tier on CI services, where we can only have max 5 jobs running in parallel.

With these restrictions in mind, I'd say it's more important to ensure that PL works with a variety of Python versions as well as the latest release versions of TensorFlow/PyTorch; users having issues can more easily upgrade TF/PyTorch than upgrade Python.